src/components/uploader/css/Uploader.css

```diff
@@ -241,7 +241,7 @@ tr.fl-uploader-file-state-uploaded button:before {
     display: block;
     overflow: hidden;
     height: 2em;
-    width: 7em;
+    width: 9em;
```

Comment:
Does it have to be all the to `9em`? There appears to be a lot of extra space to the right at this size.